policies/attestation_task_bundle.rego

```diff
@@ -0,0 +1,36 @@
+package policies.attestation_task_bundle
+
+import data.lib
+
+allowed_bundles := ["quay.io/appstudio-utils:latest"]
+
+# METADATA
+# title: Task bundle was not used or is not defined
+# description: |-
+#   Check for existence of a task bundle. Enforcing this rule will
+#   fail the contract if the task is not called from a bundle.
+# custom:
+#   short_name: disallowed_task_reference
+#   failure_msg: Task '%s' does not contain a bundle reference
+deny[result] {
+	task := lib.tasks[_]
+	name := task.name
+	not task.ref.bundle
```

Comment:
Will this catch an empty string as well? Maybe a unit tests is helpful here.

Reply:
This rule doesn't catch it, but the second rule does. Is that good enough or is there value having this rule catch it also? I did add a test for it just now.

Reply:
Nah. All good.